WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -223,8 +222,12 @@
         sign_in delegate
       end
 
-      it 'creates a new competition' do
-        post :create, params: { competition: { name: "Test 2015", delegate_ids: delegate.id } }
+      it 'creates a new competition with organizers and expect them to receive a notification email' do
+        organizer = FactoryBot.create :user
+        expect(CompetitionsMailer).to receive(:notify_organizer_of_addition_to_competition).and_call_original
```

Comment:
I can't call a competition in the CompetitionsMailer which gets defined by `new_comp` afterwards which is why I did not add a `with` condition here.